packages/docs/src/components/block.tsx

```diff
@@ -4,17 +4,16 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
-import { IPageData, StringOrTag } from "documentalist/dist/client";
+import { IBlock, IPageData } from "documentalist/dist/client";
```

Comment:
we still have to reach into the package submodules? :(

Reply:
@adidahiya there's a very good reason for this: importing from the main `documentalist` module brings in _everything_: typescript, kss, etc, all of which fail on importing `fs` because it tries to load a node library in a browser environment. **this `client` submodule only exports interfaces and util functions like type guards.** 

it's critical to provide a separation here as a number of entities are necessary for use in a theme, and a number of other entities are explicitly not to be used in theme rendering (like `TypescriptPlugin`).

**def looking for a better solution here!** an easy workaround is to remove the `export * from "./client"` in the index file so you can actually _only_ get these symbols from `dist/client`. typescript & vs code are smart enough to resolve the symbols to the submodule.

Reply:
@adidahiya: I did the workaround above here https://github.com/palantir/documentalist/pull/56/files#diff-f41e9d04a45c83f3b6f6e630f10117feL8